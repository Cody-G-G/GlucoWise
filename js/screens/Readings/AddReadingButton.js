'use strict';
import React, {Component} from 'react';
import {View, TouchableOpacity} from 'react-native';
import {Icon} from 'native-base';
import styles from "./styles";

export default class AddReadingButton extends Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <TouchableOpacity onPress={this.props.addReading} style={styles.addReadingButton}>
                <Icon theme={{iconFamily: "MaterialIcons"}} style={styles.addReadingButtonIcon} name="add"/>
            </TouchableOpacity>
        );
    }
}