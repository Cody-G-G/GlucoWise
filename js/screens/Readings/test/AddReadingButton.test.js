'use strict';
import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';
import AddReadingButton from "../AddReadingButton";

test('AddReadingButton - renders correctly', () => {
    const tree = renderer.create(
        <AddReadingButton/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});