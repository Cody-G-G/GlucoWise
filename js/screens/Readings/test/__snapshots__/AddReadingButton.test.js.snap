// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddReadingButton - renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "forestgreen",
      "flex": 1,
      "justifyContent": "center",
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "white",
          "fontSize": 40,
        },
        Object {
          "fontFamily": "Material Icons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
    theme={
      Object {
        "iconFamily": "MaterialIcons",
      }
    }
  >
    î……
  </Text>
</View>
`;
